kinto/__main__.py

```diff
@@ -46,6 +47,17 @@ def main(args=None):
     parser_migrate = subparsers.add_parser('migrate')
     parser_migrate.set_defaults(which='migrate')
 
+    parser_delete_collection = subparsers.add_parser('delete-collection')
+    parser_delete_collection.add_argument(
+        '--bucket',
+        help='The bucket the collection is in.',
```

Comment:
We discussed this on IRC. "The bucket the collection is in" sounds fine to me, but if you don't like it, some other alternatives include "The bucket where the collection is", "The bucket with the collection in it", "The bucket containing the collection", or "The bucket that contains the collection".